lms/djangoapps/verify_student/migrations/0010_auto__del_field_softwaresecurephotoverification_window.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Deleting field 'SoftwareSecurePhotoVerification.window'
+        db.delete_column('verify_student_softwaresecurephotoverification', 'window_id')
+
+
+    def backwards(self, orm):
+        # Add field 'SoftwareSecurePhotoVerification.window'. Setting its default value to None
+        if db.backend_name == 'mysql':
+            db.execute('ALTER TABLE verify_student_softwaresecurephotoverification ADD `window_id` int(11) DEFAULT NULL;')
```

Comment:
This won't do anything if the database backend isn't mysql.  Should there be an `else` statement here to recreate the column using the South API?